pkg/cmd/create/create.go

```diff
@@ -490,7 +490,7 @@ func (o *Options) findPreviewURL(envVars map[string]string) (string, error) {
 				var err error
 				url, _, err = kserving.FindServiceURL(ctx, o.KServeClient, o.KubeClient, releaseNamespace, n)
 				if err != nil {
-					return errors.Wrapf(err, "failed to ")
+					log.Logger().Debugf("failed to ", err)
```

Comment:
can we do warnf? Debug logs wont be shown unless someone uses debug mode.
Also use format specifier when using `debugf`.
Would be good to give more context around this error.

Reply:
okay sure : )